.changes/next-release/feature-625c1f2c-3dfd-47cb-8907-408041a3642c.json

```diff
@@ -0,0 +1,4 @@
+{
+  "type" : "feature",
+  "description" : "Uses AB variation as the name for overriden customizations"
```

Comment:
can omit because not directly customer facing

Reply:
We didn't really have a contract for how we plumb this additional data through so marked it as a feature because it's new but could go either way